packages/samples/stencil/stencil.config.ts

```diff
@@ -36,6 +38,36 @@ export const config: Config = {
       html({
         include: 'src/components/**/*.html',
       }),
+      resolveAtomicPaths(),
     ],
   },
 };
+
+function resolveAtomicPaths() {
+  return {
+    resolveId(source: string) {
+      if (source.startsWith('@coveo/atomic/')) {
+        const nodeModulesLocation = '../../../node_modules';
+
+        const packageJsonPath = join(
+          process.cwd(),
+          nodeModulesLocation,
+          '@coveo',
```

Comment:
You could probably use require.resolve or import.resolve to get the base URL! :)

Reply:
I tried and I was having trouble with it since the node_modules are at the root ! I think this is fine for now